utils/run_test.sh

```diff
@@ -68,6 +68,15 @@ run_test()
     fi
 
     ((log_num += 1))
+
+    FILES=${DAOS_BASE}/test_results/*
```

Comment:
(lint) Brace expansions and globs are literal in assignments. Quote it or use an array. [SC2125]